frontend/webpack.config.js

```diff
@@ -44,18 +44,26 @@ module.exports = (_env, argv) => ({
             }],
         }, {
             test: /\.css$/u,
-            loader: "raw-loader",
+            type: "asset/source",
         }],
     },
 
     plugins: [
         new CleanWebpackPlugin(),
-        new HtmlWebpackPlugin({
-            inject: false,
-            template: path.join(APP_PATH, "index.html"),
+        new ForkTsCheckerWebpackPlugin({
+            eslint: {
+                files: ["*.js", "src"],
```

Comment:
```suggestion
                files: ["."],
```

To be consistent with our other ESLint invocations.

Reply:
Mh ok. I explicitly wrote it out like that because it was noticeably slower with `"."` before. But I cannot reproduce that anymore. Maybe I tested that before using the cache or something like that.